File: CRUD/PHP/Symfony3 V1/Symfony3/src/Sistema/Bundle/Services/Funciones.php
<?php

// Written By Ismael Heredia in the year 2017

namespace Sistema\Bundle\Services;

class Funciones
{

	public function mensaje($titulo,$contenido,$tipo) {
		return "
		<script>
		swal({
				title: '".$titulo."',
				text: '".$contenido."',
				type:'".$tipo."',
				html:true,
				animation: false
	     });
		</script>
		";
	}

	public function mensaje_con_redireccion($titulo,$contenido,$tipo,$ruta) {
		return "
		<script>
		swal({
				title: '".$titulo."',
				text: '".$contenido."',
				type:'".$tipo."',
				html:true,
				animation: false
	     },function() {
	        window.location.href = '".$ruta."';  
	     });
		</script>
		";
	}

	public function fecha_actual() {
		date_default_timezone_set("America/Argentina/Cordoba");
		$fecha = date("d/m/Y", time());
		return $fecha;
	}

}

File: CRUD/C#/Aplicacion Web sobre Personal con SQL Server/Personal/Personal/Functions/Funciones.cs
// Written By Ismael Heredia in the year 2017

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Personal
{
    public class Funciones
    {
        public string mensaje(string titulo, string contenido, string tipo)
        {
            return "<script>" +
                    "swal({" +
                            "title: '" + titulo + "'," +
                            "text: '" + contenido + "'," +
                            "type:'" + tipo + "'," +
                            "html:true," +
                            "animation: false" +
                     "});" +
                    "</script>";
        }
    }
}